[docs] Simplify README (#18970)
web-platform-tests/wpt: resources/readme.md

# Resources

> This directory contains utilities intended for use by tests.

Reviewer: ```suggestion
This directory contains infrastructure resources intended for use by tests.
```

> This directory contains utilities intended for use by tests of the web

Reviewer: ```suggestion
This directory contains utilities intended for use by tests.
```

> platform. It does not contain tests for web platform features.

Reviewer 2: I think "of the web platform" and "for web platform features" seems redundant.

> platform. It does not contain tests for web platform features.

Reviewer: ```suggestion
It does not contain tests.
```

## `testharness.js`

`testharness.js` is a framework for writing low-level tests of
browser functionality in javascript. It provides a convenient API for
making assertions and is intended to work for both simple synchronous
tests, and tests of asynchronous behaviour.

Complete documentation is available in the `docs/` directory of this repository

> and on the web at http://web-platform-tests.org/writing-tests.

Reviewer: ```suggestion
and on the web at https://web-platform-tests.org/writing-tests/.
```
